rules/ppt-test-please/rule.md

```diff
@@ -3,20 +3,30 @@ type: rule
 archivedreason: 
 title: Do you get your "PPT Tester" to do a 'Test Please' (preferably a designer)?
 guid: 44956034-2782-47c8-9165-c8ccf514b4a4
-uri: do-you-get-your-ppt-tester-to-do-a-test-please-preferably-a-designer
+uri: ppt-test-please
 created: 2010-06-08T10:08:01.0000000Z
-authors: []
-related: []
+authors: 
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+  - title: Tiago Araujo
+    url: https://ssw.com.au/people/tiago-araujo
+related: 
+- conduct-a-test-please-internally-and-then-with-the-client
 redirects:
 - do-you-get-your-＂ppt-tester＂-to-do-a-test-please-(preferably-a-designer)
 - do-you-get-your-ppt-tester-to-do-a-test-please-(preferably-a-designer)
+- do-you-get-your-ppt-tester-to-do-a-test-please-preferably-a-designer
 
 ---
 
-Image is everything. Improve the way you market, educate and promote yourself with striking presentations. Your content might be great - but if it is displayed poorly - it will be overshadowed by its flaws. Use a designer to make your presentation stand out.   
+Image is everything. Improve the way you market, educate and promote yourself with striking presentations. Your content might be great - but if it is displayed poorly - it will be overshadowed by its flaws. Use a designer to make your presentation stand out.  
+
 <!--endintro-->
 
-## Related links
+1. Do the best job you can on your presentation
+2. Get a senior developer to go through it
+3. ⭐️ Then give to a designer for better look and feel
+4. Show it to an audience to get feedback
 
-[Do you conduct a "test please" internally and then with the client?](/conduct-a-test-please-internally-and-then-with-the-client)
-![](before_after.jpg)**Figure: Before and after... Designers can make anything look good**
+\[TODO: Hashem - Add a before and after image \]
```

Comment:
Cc: @hashemelhossainissw 
(Hoping this is the correct @) 
TODO:  ADD IMAGES 😎

Just making sure Hashem is aware

Reply:
Hey @JackDevAU thanks for the heads up.

Yeap, there is an email and we also spoke on Teams about this.

I have merged this change because we don't want the current photo live anymore. 

Hey @hashemelhossainissw,

Please generate a new PR for the new photo ;)